src/main/java/com/epam/reportportal/junit/ReportPortalListener.java

```diff
@@ -582,6 +593,12 @@ protected void stopTestMethod(@Nonnull final Object runner, @Nonnull final Frame
 	protected void stopTestMethod(@Nonnull final Object runner, @Nonnull final FrameworkMethod method,
 			@Nonnull final ReflectiveCallable callable, @Nonnull final ItemStatus status, @Nullable final Throwable throwable) {
 		FinishTestItemRQ rq = buildFinishStepRq(runner, method, callable, status);
+		if (status != ItemStatus.PASSED && throwable != null) {
+			rq.setDescription(String.format(DESCRIPTION_TEST_ERROR_FORMAT,
+					createStepDescription(this.context.getTestMethodDescription(method), method),
+					throwable.getClass().getSimpleName(),
+					ofNullable(throwable.getMessage()).isPresent() ? throwable.getMessage() : ""));
```

Comment:
Las error log contains StackTrace, not just error message. Please put it into the description. Please use `ExceptionUtils.removeCommonFrames` method to simplify Stack Trace, but putting just message is not informative.

Reply:
This type of message I confirmed with Andrey. I changed it to the full StackTrace.